<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EELEditor</class>
 <widget class="QMainWindow" name="EELEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EEL Script Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>18</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_file"/>
    <addaction name="actionSave"/>
    <addaction name="actionClose_filw"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionFind_Replace"/>
    <addaction name="actionGo_to_line"/>
   </widget>
   <widget class="QMenu" name="menuNavigation">
    <property name="title">
     <string>Navigation</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionJump_to_function"/>
    <addaction name="separator"/>
    <addaction name="actionGo_to_init"/>
    <addaction name="actionGo_to_sample"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionEEL2_documentation"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuNavigation"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen_file">
   <property name="text">
    <string>Open File</string>
   </property>
   <property name="toolTip">
    <string>Open file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose_filw">
   <property name="text">
    <string>Close File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionFind_Replace">
   <property name="text">
    <string>Find/Replace...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionGo_to_line">
   <property name="text">
    <string>Go to Line...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionJump_to">
   <property name="text">
    <string>Jump to...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="actionJump_to_function">
   <property name="text">
    <string>Jump to function...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionPrevious_function">
   <property name="text">
    <string>Previous function...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+M</string>
   </property>
  </action>
  <action name="actionGo_to_init">
   <property name="text">
    <string>Go to @init</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionGo_to_sample">
   <property name="text">
    <string>Go to @sample</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionAutomatic_Parentheses">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Automatic Parentheses</string>
   </property>
  </action>
  <action name="actionReplace_Tabs_with_Spaces">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Replace Tabs with Spaces</string>
   </property>
  </action>
  <action name="actionEEL2_documentation">
   <property name="text">
    <string>EEL2 documentation</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
